wsgi/odyssy/announcement/api/urls.py

```diff
@@ -0,0 +1,15 @@
+from django.conf.urls import url
+
+from .views import (
+    AnnouncementListView,
+    AnnouncementParticularView,
+    AnnouncementTagView,
+    AnnouncementLatestView,
+)
+
+urlpatterns = [
+    url(r'^$', AnnouncementListView.as_view(),  name='all-announcement'),
```

Comment:
It would be more relevant if it would be `api_all_announcement`. View @Monal5031